runtime/src/accounts_db.rs

```diff
@@ -266,6 +266,21 @@ impl AccountStorageEntry {
         self.approx_store_count.load(Ordering::Relaxed)
     }
 
+    // .len()/.is_empty()/.capacity() is mimicing std::collection api
+    pub fn len(&self) -> u64 {
```

Comment:
well, this is in interms of bytes; maybe, mimicing the api doesn't make sense; I'll rename later.